confignode/src/main/java/org/apache/iotdb/confignode/manager/ConsensusManager.java

```diff
@@ -48,9 +48,8 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(ConsensusManager.class);
   private static final ConfigNodeConf conf = ConfigNodeDescriptor.getInstance().getConf();
 
-  private IConsensus consensusImpl;
-
   private ConsensusGroupId consensusGroupId;
+  private IConsensus consensusImpl;
 
   private DeviceGroupHashExecutor hashExecutor;
```

Comment:
Why we change the DeviceGroup name in the conf but not in the code?

Reply:
fixed.